Source/UI/Controllers/CardViewController.swift

```diff
@@ -177,9 +179,18 @@ public class CardViewController: UIViewController,
     }
 
     @objc func onTapAddressView() {
-        navigationController?.pushViewController(addressViewController, animated: true)
-        checkoutAPIService?.logger.log(.billingFormPresented)
-        suppressNextLog = true
+
+        defer {
+            suppressNextLog = true
+            checkoutAPIService?.logger.log(.billingFormPresented)
+        }
+        guard isNewUI,
+                let viewController = BillingFormFactory.getBillingFormViewController(delegate: self).1 else {
```

Comment:
is there something called first ? `BillingFormFactory.getBillingFormViewController(delegate: self).1 -> BillingFormFactory.getBillingFormViewController(delegate: self).first  `

Reply:
I have checked, there's no `.first` for tuples

Reply:
if you label the elements of the tuple returned by `getBillingFormViewController` then you could do `BillingFormFactory.getBillingFormViewController(delegate: self).viewController`